syntax = "proto3";

package monitoring.v1;

option go_package = "github.com/smnzlnsk/monitoring-proto-lib/monitoring/v1";

message NotifyDeletionRequest {
  string job_name = 1;
  int32 instance_number = 2;
}

message NotifyDeletionResponse {
  bool acknowledged = 1;
  string message = 2;
}